src/components/MARCTableView/MARCTableViewRow.js

```diff
@@ -36,8 +38,14 @@ export const MARCTableViewRow = ({
     subaction,
   } = rowData;
 
+  const getCellValue = value => {
+    const formattedValue = value && value.trim ? value.trim() : value;
```

Comment:
I see this code is repeated in several places. Consider moving this to `utils`.

Reply:
agree!